server/main.lua

```diff
@@ -36,13 +40,14 @@ RegisterNetEvent('qb-streetraces:JoinRace', function(RaceId)
             if xPlayer.Functions.RemoveMoney('cash', Races[RaceId].amount, "streetrace-joined") then
                 TriggerClientEvent('qb-streetraces:SetRace', -1, Races)
                 TriggerClientEvent('qb-streetraces:SetRaceId', src, RaceId)
+                TriggerClientEvent('qb-streetraces:BeginPhasing', src)
```

Comment:
Not sure what this points too, need to remove.

Reply:
Phasing in street racing in GTARP is removing collision between racers. I use it in custom version. This is a remnant. Removed.